src/api/apiFetch.js

```diff
@@ -11,10 +9,6 @@ const apiVersion = 'api/v1';
 export const apiFetch = async (auth: Auth, route: string, params: Object = {}) => {
   const url = `${auth.realm}/${apiVersion}/${route}`;
 
-  if (!isUrl(url)) {
```

Comment:
Removing the url validation from here is a regression.
Even if you are replacing the code you should have kept the call to the new function.